pom.xml

```diff
@@ -65,6 +65,9 @@
         <buildnumber-maven-plugin.version>3.2.1</buildnumber-maven-plugin.version>
         <exec-maven-plugin.version>3.5.0</exec-maven-plugin.version>
         <cyclonedx-maven-plugin.version>2.9.1</cyclonedx-maven-plugin.version>
+        <!-- Graalvm libs -->
+        <graalvm.sdk.version>17.0.12</graalvm.sdk.version>
```

Comment:
I think we can get rid of: `graalvm.sdk.version` and keep a single: `graalvm.version`:23.0.5 ?